docs/concepts/usage-ideas-examples.md

```diff
@@ -2,14 +2,317 @@
+
+### Decentralized DNS
+
+The [Domain Name System](https://en.wikipedia.org/wiki/Domain_Name_System) (DNS) is one of the most centralized pieces of the web. By having to ask a central place to know where to find  `google.com` or `facebook.com`, it is a central point of failure. Backing up the DNS on IPFS could improve its availability. A prototype for Javascript has been [published on NPM](https://www.npmjs.com/package/orbitdns).
+
+### Archive the web
+
+Thanks to de-duplication, IPFS is a powerful tool for archiving the web. The [InterPlanetary Wayback](https://github.com/oduwsdl/ipwb) is tackling this endeavor.
+
+### Fight censorship
+
+Wikipedia has not been accessible in Turkey for a few years. The company behind IPFS, Protocol Labs, is hosting a mirror of Wikipedia on IPFS. See the original [blog post](https://blog.ipfs.io/24-uncensorable-wikipedia/) and the [project code](https://github.com/ipfs/distributed-wikipedia-mirror) for more information.
+
+### Alternative ISP
+
+[Althea](https://althea.net/about) is building an alternative [Internet Service Provider](https://en.wikipedia.org/wiki/Internet_service_provider) (ISP) in the US as the low level of competition brings the quality of service down and prices up. Althea especially targets rural areas, where the coverage by traditional ISP is low.
+
+## Blockchain use-cases
+
+IPFS is a natural fit for [blockchain](https://en.wikipedia.org/wiki/Blockchain) use cases. The common state of the chain is distributed on-chain among participants, and specific data is stored on IPFS. Thanks to content addressing, the blockchain only needs to store the IPFS multi-hash, and users are sure to fetch correct data from any of their peers. This architecture is becoming the de-facto standard for blockchain applications.
+
+### Global market place for data storage
+
+[Filecoin](https://filecoin.io/) lets any storage owner host data for users who need additional space. It is one of the reasons IPFS exists in the first place and will be a great accelerator of the IPFS growth once fully online later this year. The project is developed by [Protocol Labs](https://protocol.ai/).
+
+Textile.io is building the [Powergate](https://blog.textile.io/filecoin-developer-tools-concepts/) tool for your app to interact with Filecoin once the main net is launched.
+
+### Reconcile IoT and privacy
+
+The [Iotex](https://www.iotex.io/) blockchain wants to build an ecosystem where Internet-of-Things (IoT) data is owned by the owners of those devices. Data from hardware devices such as cameras can be stored on local hardware, backed up to cloud, or to IPFS. Future integration with Filecoin will further reinforce the ownership of data by the users.
+
+Tangle is a distributed ledger of the [IOTA](https://www.iota.org/) network, and it supports hosting IoT data on IPFS. You can find out more [on GitHub](https://github.com/iotaledger/poc-ipfs).
```

Comment:
IOTA uses "the Tangle", not "Tangle". The Tangle is another form of distributed ledger, just like a blockchain. So I think the Tangle should be used like the (Bitcoin) blockchain.

Reply:
I see, again looks like I've misunderstood. Can you please expand on _The Tangle_ and what it's for.

Reply:
Done